src/css/tabs/static_tab.css

```
@@ -1,6 +1,6 @@
 #tab-static {
     background-color: inherit;
-    background-image: url(/images/osd-bg-1.jpg);
+    background-image: url(../../images/osd-bg-1.jpg);
```

Stylelint (17.6.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)